docs/releasing/publishing.md

```diff
@@ -1,17 +1,33 @@
-# Publishing GOV.UK Frontend
+# Steps you must complete before you release a new version of GOV.UK Frontend
 
-1. Make sure you have cards in the "Backlog" column to:
-  - Update the GOV.UK Design System to use the latest release
-  - Update the GOV.UK Prototype Kit to use the latest release
-  - Update the GOV.UK Frontend Docs to use the latest release
+1. Raise new issues in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-prototype-kit](https://github.com/alphagov/govuk-prototype-kit)) to:
+  - create release notes for the new release (example card: [#1986](https://github.com/alphagov/govuk-frontend/issues/1986))
+  - create release notes for the new version of GOV.UK Prototype Kit (example card: [#958](https://github.com/alphagov/govuk-prototype-kit/issues/958))
+  - create a card for the new release of GOV.UK Frontend (example card: [#1987](https://github.com/alphagov/govuk-frontend/issues/1987))
+  - update the GOV.UK Design System to use the new release of GOV.UK Frontend (example card: [#1347](https://github.com/alphagov/govuk-design-system/issues/1347))
+  - update the GOV.UK Frontend Docs to use the new release of GOV.UK Frontend (example card: [#75](https://github.com/alphagov/govuk-frontend-docs/issues/75))
+  - create a card for the new release of GOV.UK Prototype Kit (example card: [#917](https://github.com/alphagov/govuk-prototype-kit/issues/917))
```

Comment:
```suggestion
- update the GOV.UK Prototype Kit to use the new release (example card: [#923](https://github.com/alphagov/govuk-prototype-kit/issues/923))
```

Reply:
This suggestion is based on @vanitabarrett's comment above re reversing the order of lines 9 and 10, as you update the Prototype Kit before you release it.